### PR TITLE
Create relative links to files widgets

### PR DESCRIPTION
**Pull request by vladb55**

### CODE REVIEW

File: backend/app/utils/widgetLink.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+const getWidgetLink = function(filename, extension) {
```
Comment (maxmalov) [inline backend/app/utils/widgetLink.js]: Consider using function declaration for non anonymous functions

File: backend/app/utils/widgetLink.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    return path.resolve(__dirname + '/../widgets/' + filename + '/' + filename + extension);
```
Comment (maxmalov) [inline backend/app/utils/widgetLink.js]: How about windows support?

File: backend/app/utils/widgetLink.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+};
+
+module.exports = {
+    getTemlateLink: function(filename) {
```
Comment (maxmalov) [inline backend/app/utils/widgetLink.js]: You can shortcut it to

```js
getTemplateLink(filename) {
  // ...
}
```

File: backend/app/utils/helpers/widgetLink.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+const getWidgetLink = function(filename, extension) {
+    return path.resolve(__dirname + '/../widgets/' + filename + '/' + filename + extension);
+};
+
+module.exports = {
+    getTemlateLink: function(filename) {
```
Comment (102) [inline backend/app/utils/helpers/widgetLink.js]: `getTemplateLink`?

**Pull request merged**